src/components/photography/slider.js

```diff
@@ -11,6 +11,8 @@ const _ = {
   map,
 }
 
+const DEFAULT_INDICATOR_COLOR = 'rgb(113, 113, 113)'
```

Comment:
Since we mostly use camelCase to represent constants in this repo, would you mind renaming it to `defaultIndicatorColor` here ?

Reply:
okok

Reply:
fixed, please have a look, thanks!